Makefile

```diff
@@ -28,3 +28,10 @@ clean-roles:
 
 ansible-galaxy:
 	ansible-galaxy install -r requirements.yml --force
+
+import-gpg-keys:
+	$(foreach var,$(shell cat gpg.recipients | awk -F: '{print $$1}'),gpg --keyserver hkp://pgp.mit.edu --search-keys $(var);)
+
+recrypt:
+	ansible-vault decrypt group_vars/all/secure && pwgen -cynC1 15 | gpg --batch --yes --trust-model always -e -o vault_passphrase.gpg $(shell cat gpg.recipients | awk -F: {'printf "-r "$$1" "'}) && ansible-vault encrypt group_vars/all/secure
```

Comment:
The Makefile has now been modified so that the import step is always run before the recrypt one.